incubator/kafqa/values.yaml

```diff
@@ -0,0 +1,73 @@
+# Default values for kafqa.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+environment: "integration"
+appDurationMs: "5000000000"
+acks: "1"
+appPort: 8081
+namespace: "hermes"
+
+kafka:
+  cluster: "test"
+  topic: "kafqa-test-topic"
+
+producer:
+  enabled: "false"
+  kafkaBrokers: "localhost:9092"
+  totalMessages: -1
+  concurrency: 1
+  workerDelayMs: 50
+
+consumer:
+  enabled: false
+  kafkaBrokers: "localhost:9092"
+  groupId: "consumer-test-001"
+  concurrency: 1
+  pollTimeoutMs: 300
+  enableAutoCommit: true
+
+librd:
+  enabled: false
+  queueMaxSize: 10000
+  batchSize: 10000
+  queuedMinMessages: 30000
+  statsIntervalMs: 500
+
+prometheus:
+  enabled: true
+  port: 9090
+  url: "https://localhost:8080/v1/prom/metrics"
+  token: "test"
+
+stats:
+  enabled: true
+  host: "localhost"
+  port: 8125
+
+image:
+  repository: gojektech/kafqa
+  tag: latest
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
```

Comment:
add resource limits here and use the variables in template instead of hardcoding

Reply:
Sure I will add.